web/src/beta/features/AssetsManager/item/AssetListItem.tsx

```diff
@@ -44,15 +60,32 @@ const AssetListItem: FC<AssetItemProps> = ({
           size={20}
         />
       </Thumbnail>
-      <AssetName>
-        <Typography size="body">{asset.name}</Typography>
-      </AssetName>
-      <Col width={30}>
+      <Col width={40}>
+        <AssetName>
+          <Typography size="body">{asset.name}</Typography>
+        </AssetName>
+      </Col>
+      <Col width={20}>
         <Typography size="body">{formattedDate}</Typography>
       </Col>
-      <Col width={30}>
+      <Col width={10}>
         <Typography size="body">{formattedSize}</Typography>
       </Col>
+      <Col
+        otherProperties={{ display: "flex", alignItems: "center" }}
```

Comment:
Maybe we can add these two properties to `Col` directly?

Reply:
Thanks. Yes we can.